src/VisualStudio/TestUtilities2/CodeModel/CodeModelTestState.vb

```diff
@@ -86,6 +86,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel
 
             If Not Me._disposedValue Then
                 If disposing Then
+                    VisualStudioWorkspace.Dispose()
                     Workspace.Dispose()
```

Comment:
we have two separate workspaces?

Reply:
Yes, this is a poorly written set of tests.

Reply:
Can we just fix it?

Reply:
Not easily. `TestWorkspace` uses inheritance but not from `VisualStudioWorkspace`. I looked at this in the past but it didn't seem like a high-value win and was a lot of work. It primarily occurs in `CodeModel` tests.